### PR TITLE
Added trigrams assignment

### PR DESCRIPTION
**Pull request by esokada**
Thanks for looking at this!

### CODE REVIEW

File: students/esokada/lesson04/trigrams.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import random
+
```
Comment (charlesrobison) [inline students/esokada/lesson04/trigrams.py]: Add another blank line after import statements

File: students/esokada/lesson04/trigrams.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        for line in f:
+            #Eliminate everything above the Table of Contents, all the chapter headings, and the footer.
+            if "Contents" in line:
+                startpoint = True
+                continue
+            if "CHAPTER" in line:
+                continue
+            if "End of Project Gutenberg's" in line:
+                break
+            if startpoint == True:
```
Comment (charlesrobison) [inline students/esokada/lesson04/trigrams.py]: Should this be if startpoint is is True:, or if startpoint:?

File: students/esokada/lesson04/trigrams.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    for i in range(len(words)-2):
+        pair = words[i:i+2]
+        follower = words[i+2]
+        #If the pair of words is already a key in the dictionary, add the follower to it. Otherwise, add a new key.
+        if tuple(pair) in trigrams:
+            trigrams[tuple(pair)].append(follower)
+        else:
+            trigrams.update({tuple(pair):[follower]})
+    return trigrams
+
```
Comment (charlesrobison) [inline students/esokada/lesson04/trigrams.py]: PEP8 formatting here too - add another blank line between functions

File: students/esokada/lesson04/trigrams.py (hunk 2 above)
Comment (esokada) [reply students/esokada/lesson04/trigrams.py]: Right, thanks! I've got to kick this bad habit.  :)

**Comment by esokada** [comment]
Thanks very much for your comments, and I hope you enjoy your vacation!

**Pull request merged**